src/main/kotlin/io/Ust.kt

```diff
@@ -192,17 +219,46 @@ object Ust {
             line.tryGetValue("VBR")?.let {
                 pendingVBR = it.split(',').mapNotNull { cell -> cell.toDoubleOrNull() }
             }
+            line.tryGetValue("Piches")?.let {
+                if (pendingNotePitches != null)
+                    return@let
+                pendingNotePitches = parseMode1PitchData(it)
+            }
+            line.tryGetValue("Pitches")?.let {
+                if (pendingNotePitches != null)
+                    return@let
+                pendingNotePitches = parseMode1PitchData(it)
+            }
+            line.tryGetValue("PitchBend")?.let {
+                if (pendingNotePitches != null)
+                    return@let
+                pendingNotePitches = parseMode1PitchData(it)
+            }
+        }
+        val pitchDataMode1 = UtauMode1TrackPitchData(notePitchDataListMode1)
+        val pitchDataMode2 = notePitchDataListMode2.ifEmpty { null }?.let { UtauMode2TrackPitchData(it) }
+        return FileParseResult(file, projectName, notes, tempos, isMode2, pitchDataMode1, pitchDataMode2)
+    }
+
+    private fun parseMode1PitchData(
```

Comment:
In mode2 process, I used `UtauMode2TrackPitchData` to contain all the RAW data, which means we do not do any conversion. Please check `UtauMode2TrackNoteData` and you will see it's totally raw.
The process is done at the next step: https://github.com/sdercolin/utaformatix3/pull/36/files#diff-ece6cb33a5e5ea3b42655bdfa95ae32cb7422707a203999cf0e674f46d5dffe0R57

So to keep consistency, in this function we should only do `split()` and type cast.